sound-equipment-fault-detection/vled/control/control.go

```diff
@@ -0,0 +1,59 @@
+package control
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/kubeedge/examples/sound-equipment-fault-detection/vled/device"
+	"k8s.io/klog/v2"
```

Comment:
ditto, sort the import. Please fix the order of other similar files as well.

Reply:
Thanks, I have completed the adjustment.